vcd/resource_vcd_org_vdc.go

```diff
@@ -20,43 +20,24 @@ func resourceVcdOrgVdc() *schema.Resource {
 		MaxItems: 1,
 		Elem: &schema.Resource{
 			Schema: map[string]*schema.Schema{
-				"units": {
-					Type:         schema.TypeString,
-					Required:     true,
-					ForceNew:     true,
-					ValidateFunc: validation.StringInSlice([]string{"MHz", "GHz", "MB", "GB"}, false),
-					Description:  "Units in which capacity is allocated. For CPU capacity, one of: {MHz, GHz}.  For memory capacity, one of: {MB, GB}.",
-				},
 				"allocated": {
 					Type:        schema.TypeInt,
-					Optional:    true,
+					Required:    true,
```

Comment:
I'm not yet certain this is needed in Pay-as-you-go, discussing.

Reply:
API if isn't using, it still requires it.

Reply:
Let's think how to resolve this, right now it's very confusing because in AllocationVApp even zero is not accepted, while it's not used at all, so you end up providing a dummy value.

Reply:
fixed